app/policies/service_provider_mfa_policy.rb

```diff
@@ -1,5 +1,9 @@
 class ServiceProviderMfaPolicy
-  PHISHING_RESISTANT_METHODS = %w[webauthn webauthn_platform piv_cac].freeze
+  PHISHING_RESISTANT_METHODS = [
+    TwoFactorAuthenticatable::AUTH_METHOD_WEBAUTHN,
+    TwoFactorAuthenticatable::AUTH_METHOD_WEBAUTHN_PLATFORM,
+    TwoFactorAuthenticatable::AUTH_METHOD_PIV_CAC,
+  ].freeze
```

Comment:
if we did the module/namespace approach, we could move this to that module. `AuthMethod.phishing_resistant?(value)`

Reply:
I made an attempt at this in [a4ebf5f](https://github.com/18F/identity-idp/pull/8315/commits/a4ebf5ff356ddedf887b8c323314903fd2cf8761) (alongside the suggestion [here](https://github.com/18F/identity-idp/pull/8315#discussion_r1181800769))